scripts/ts-utils/setup-enclave.ts

```diff
@@ -66,20 +95,21 @@ async function updateScheduledEnclave(api: any, signAccount: any) {
 
 async function main() {
 
+    // NOTE: If we are using development environment, we can use Alice instead of moving around with keys
     const keyring = new Keyring({ type: "sr25519" });
-    let signAccount = keyring.addFromJson(private_account_pair);
+    // let signAccount = keyring.addFromJson(private_account_pair);
 
     console.log(colors.green("account unlock..."));
 
     //unlock account
-    signAccount.toJson(accountPassword);
+    // signAccount.toJson(accountPassword);
 
     const { defaultAPI } = await initApi();
     const Alice = keyring.addFromUri("//Alice", { name: "Alice default" });
 
     await transfer(defaultAPI, Alice);
-
-    await updateScheduledEnclave(defaultAPI, signAccount);
+    await setTeerexAdmin(defaultAPI, Alice);
+    await updateScheduledEnclave(defaultAPI, Alice, block);
```

Comment:
Hmm - there can be a timing issue here.

The `await` would return as soon as the tx was included in the block, but it doesn't mean the change has taken effect. The sidechain would need to sync parachain blocks to actually execute it.